packages/core/src/providers/email.ts

```diff
@@ -111,6 +111,8 @@ export function Email(config: EmailConfig): EmailConfig {
   }
 }
 
+export default Email
```

Comment:
I tried hardcoding this into my node_modules/@auth/core/src/providers/email.ts but the error persist

Reply:
ye because it should be email.js not .ts

Reply:
What else could I be missing? 
I'm sure I added that change to 
`@auth/core/providers/email.js`

Reply:
> What else could I be missing? I'm sure I added that change to `@auth/core/providers/email.js`

the issue is that i wasn't default exported so it throws this issue, i tried this using solid-start (vite as-well) and it seemed to throw the exact same issue, after adding this line to email.js this issue disappears

Reply:
> 
make sure you add this to email.js and restart the dev server.